src/input_streams.rs

```diff
@@ -56,9 +56,20 @@ impl<'a> InputStreams<'a> {
         let keycodes = world.get_resource::<Input<KeyCode>>();
         let scan_codes = world.get_resource::<Input<ScanCode>>();
         let mouse_buttons = world.get_resource::<Input<MouseButton>>();
-        let mouse_wheel = world.get_resource::<Events<MouseWheel>>();
+        let mouse_wheel = world.resource::<Events<MouseWheel>>();
         let mouse_motion = world.resource::<Events<MouseMotion>>();
 
+        let mouse_wheel: Vec<MouseWheel> = mouse_wheel
+            .get_reader()
```

Comment:
There's no way to correctly read the events from here.
Same thing for the `from_world` method on `MutableInputStreams` .